metis-transformation/metis-transformation-service/src/main/java/eu/europeana/metis/transformation/service/XsltTransformer.java

```diff
@@ -134,12 +140,19 @@ private static Templates getTemplates(String xsltUrl) throws CacheValueSupplierE
 
   private static Templates createTemplatesFromUrl(String xsltUrl)
       throws CacheValueSupplierException {
+
+    HttpRequest httpRequest = HttpRequest.newBuilder()
+        .GET()
+        .uri(URI.create(xsltUrl))
+        .build();
+
     final TransformerFactory transformerFactory = new TransformerFactoryImpl();
     // We know where the xslt files are coming from, we consider them safe.
-    try (@SuppressWarnings("findsecbugs:URLCONNECTION_SSRF_FD") final InputStream xsltStream = new URL(
-            xsltUrl).openStream()) {
+    try {
+      final InputStream xsltStream = httpClient.send(httpRequest, BodyHandlers.ofInputStream())
+          .body();
```

Comment:
The inputStream is not closed, perhaps use try with resources

Reply:
Done